Added pages to the nav bar, Finished Cuckoo Hashing example, Fixed Bst Traversal Algorithm and Example, closes #5 and #17

qsebring3930 opened the pull request.

qsebring3930 committed:
Added Navbar Pages

Added all of the proposed pages in open issues to the navigation.yml file and added their page templates.

qsebring3930 committed:
Merge branch 'mgoadric:gh-pages' into gh-pages

qsebring3930 committed:
Cuckoo hashing example finished, closes issue #5

mgoadric commented:
Nice! Can you also add in appropriate alt and caption pieces for the images?

Let's cut a few of the tree pages down in the navbar. I think we should have a heading of "Advanced Trees" with Red-Black Trees, Splay Trees, 2-3 Trees, and Huffman Coding underneath. Insertion algorithms will be fine for the first three, and creation for the Huffman, so we won't need multiple pages.

I don't think we need traversals for SkipList, but insert and remove are good.

We can leave the Disjoint Set subheadings for now, and revisit them when we get to it, that might be lower priority than the other tree algs.

qsebring3930 committed:
Tweaked nav bar, fixed Tree Traversals, closes issue #17

qsebring3930 committed:
Captions added for Cuckoo Hashing pictures

qsebring3930 commented:
Changed the nav bar to what you recommended, changed the algorithms in bst traversals to be less vague and added examples. Will update the bst traversal page with video and example pictures when I can use chits in them. Added cuckoo hashing example picture captions.
